docs/ref/contrib/sitemaps.txt

```diff
@@ -483,11 +483,24 @@ that: :func:`django.contrib.sitemaps.ping_google()`.
 
 .. function:: ping_google
 
-    :func:`ping_google` takes an optional argument, ``sitemap_url``,
-    which should be the absolute path to your site's sitemap (e.g.,
-    :file:`'/sitemap.xml'`). If this argument isn't provided,
-    :func:`ping_google` will attempt to figure out your
-    sitemap by performing a reverse looking in your URLconf.
+    :func:`ping_google` takes the following optional arguments:
+
+        * ``sitemap_url`` which should be the absolute path to your site's sitemap (e.g.,
+          :file:`'/sitemap.xml'`). If this argument isn't provided,
+          :func:`ping_google` will attempt to figure out your
+          sitemap by performing a reverse looking in your URLconf.
+
+        * ``ping_url`` defaults to `Google Webmaster Ping Tool`_.
+
+        .. _`Google Webmaster Ping Tool`: https://www.google.com/webmasters/tools/ping
+
+        .. versionadded:: 2.2
```

Comment:
This should appear below the options in a "self-contained" note as described in [Documenting new features](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features).